src/authorization/authorization.module.ts

```diff
@@ -67,6 +70,9 @@ import UserCacheService from './service/usercache.service';
     RoleService,
     RoleCacheService,
     SearchService,
+    PermissionRepository,
```

Comment:
Instead of directly providing Repository, use 
TypeOrmModule.forFeature([<Entity>,<EntityRepository>])
in imports.

Reply:
Hi @pooja-keyvalue, I've tried doing that, but it will result in exceptions like `[CustomExceptionsFilter] this.permissionRepository.getAllPermissions is not a function`